webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -157,6 +184,31 @@ const useModalModelCreate = ({ onClose }: Props = {}): ModalHooks => {
     setModalState((prev) => ({ ...prev, modelDescription: e.target.value }));
   }, []);
 
+  const onSelect = useCallback((info: SelectInfo) => {
+    setModalState((prev) => ({ ...prev, workspace: info.key }));
+  }, []);
+
+  const workspaceName = useMemo(() => {
+    const name = workspaces.find((ws) => String(ws.id) === modalState.workspace)?.name || '';
```

Comment:
you meant `??` instead of `||`?
also, if workspace name becomes `''`, user would see blank space in dropdown. that might be confusing